test_runner/src/main/kotlin/ftl/gc/GcAndroidTestMatrix.kt

```diff
@@ -56,7 +56,7 @@ object GcAndroidTestMatrix {
             .setNetworkProfile(args.networkProfile)
             .setDirectoriesToPull(args.directoriesToPull)
             .setAdditionalApks(additionalApkGcsPaths.mapGcsPathsToApks())
-            .setFilesToPush(otherFiles.mapToDeviceFiles())
+            .setFilesToPush(otherFiles.mapToDeviceFiles() + args.obbFiles.mapToDeviceFiles())
```

Comment:
`obbFiles` need to be uploaded to bucket like other files https://github.com/Flank/flank/blob/b9596bc83027a5bc94bb5d1dca9a8adb81dcea5a/test_runner/src/main/kotlin/ftl/run/platform/android/UploadOtherFiles.kt#L10

Reply:
will investigate and add whats needed.

Reply:
resolved